### ISSUE TITLE
Missing styling of the "help" button when adding ALT text

### ISSUE THREAD

**Issue by Slagt** [nileane/TangerineUI-for-Mastodon#174]
## Description of the problem

When adding alt text to a picture, the `help` button in the bottom right corner is not styled correctly.

## Example

<img width="712" height="721" alt="Image" src="https://github.com/user-attachments/assets/4706e6e6-32b7-4c5e-8afa-d01fff34d598" />

## Version

Only tested with theme Lagoon. I am using the raw CSS with user styling ([Stylus](https://github.com/openstyles/stylus) for firefox).
